webclient/core/context_processors.py

```diff
@@ -106,19 +109,15 @@ def auto_logout_client(request):
             ctx["extra_param"] = mark_safe(_("core.context_processors.autologout.expired.text"))
         ctx["logout_warning_text"] = mark_safe("AUTOLOGOUT_EXPIRATION_WARNING")
     else:
-        cache_name = str(request.user.id) + "_maintenanteLogoutTime"
-        logout_time = cache.get_or_set(
-            cache_name,
-            datetime.now() + timedelta(seconds=options["MAINTENANCE_LOGOUT_TIME"]),
-            900,
+        logger.debug("core.context_processors.auto_logout_client")
+        until_logout = datetime.combine(date.today(), odstavka_cas, pytz.timezone("Europe/Prague")) - datetime.now(
+            pytz.timezone("Europe/Prague")
         )
-        logger.debug("core.context_processors.auto_logout_client", extra={"logout_time": logout_time})
-        until_logout = logout_time - datetime.now()
-        ctx["seconds_until_idle_end"] = int(until_logout.total_seconds())
-        ctx["IDLE_WARNING_TIME"] = ctx["seconds_until_idle_end"] - 5
+        ctx["seconds_until_idle_end"] = int(until_logout.total_seconds()) - 60
+        ctx["IDLE_WARNING_TIME"] = False
         ctx["redirect_to_login_immediately"] = "logoutFunction"
         ctx["extra_param"] = mark_safe({"logout_type": "maintenance"})
-        ctx["logout_warning_text"] = mark_safe("MAINTENANCE_LOGOUT_WARNING")
+        # ctx["logout_warning_text"] = mark_safe("MAINTENANCE_LOGOUT_WARNING")
```

Comment:
Tuhle hlášku bych úplně nevynechával a 10 min před zahájením odstávky bych ji zobrazil (psalo se tam reálně "Blíží se automatické odhlášení z důvodu plánované odstávky systému. Dokončete veškerou práci a do 10 minut budete automaticky odhlášeni.", což dává smysl sdělit).

Reply:
Pridane. Spravil som to tak, aby sa to zobrazilo len raz a nie zakazdym, ked sa nacita stranka poslednych 10 minut.

Reply:
Super, díky